vertical-tabbrowser.xml

```diff
@@ -16,6 +16,40 @@
           xmlns:html="http://www.w3.org/1999/xhtml">
 
   <binding id="tabbrowser-arrowscrollbox" extends="chrome://browser/content/tabbrowser.xml#tabbrowser-arrowscrollbox">
+    <content>
+      <xul:toolbarbutton class="scrollbutton-up"
+                         xbl:inherits="orient,collapsed=notoverflowing,disabled=scrolledtostart"
+                         anonid="scrollbutton-up"
+                         onclick="_distanceScroll(event);"
+                         onmousedown="if (event.button == 0) _startScroll(-1);"
+                         onmouseup="if (event.button == 0) _stopScroll();"
+                         onmouseover="_continueScroll(-1);"
+                         onmouseout="_pauseScroll();"
+                         ondragenter="if (event.button == 0) _startScroll(-1);"
+                         ondragover="_distanceScroll(event);"
```

Comment:
Instead of copying all of this XUL, could we add the `ondragover`/`ondragleave` attributes to the elements in a constructor?